### PR TITLE
Removed integration with laminas-console and laminas-mvc-console

### PR DESCRIPTION
**Pull request by driehle**
**Feature Removal**

This PR removes the integration with laminas-console and laminas-mvc-console. Both packages have been deprecated and are not further maintained. The relevant signatures inside the DoctrineModule have been marked as deprecated since 4.2.0, now it's time to remove them.

Targeted at the next major release, i.e. 5.0.0.

### CODE REVIEW

File: src/Module.php [hunk 1]
```diff
@@ -23,8 +23,6 @@
  */
 class Module implements ConfigProviderInterface, InitProviderInterface, BootstrapListenerInterface
 {
-    use GetConsoleUsage;
-
     /** @var ServiceLocatorInterface */
     private $serviceManager;
```
Comment (snapshotpl) [inline src/Module.php]: After remove `GetConsoleUsage` can be removed

**Review by snapshotpl — CHANGES_REQUESTED** [review]

File: src/Module.php (hunk 1 above)
Comment (driehle) [reply src/Module.php]: Hm, the file src/GetConsoleUsage.php is already removed, or what do you mean?

Comment (SenseException) [reply src/Module.php]: The property doesn't seem to be in use anymore. And what about `onBootstrap()`?

Comment (driehle) [reply src/Module.php]: Ah, I see. Fixed with 4f5dcd9.

**Review by TomHAnderson — APPROVED** [review]

**Pull request merged**